### PR TITLE
Formats the project with janestreet ocamlformat settings

### PR DESCRIPTION
**Pull request by joseferben**
I find code a bit easier to scan, as things are more vertical than horizontal. I have tried it out on Sihl already and I found it improved readability.

### CODE REVIEW

**Comment by joseferben** [comment]
Review https://github.com/oxidizing/letters/pull/31 first please

File: lib/ca_certs.ml [hunk 1]
```diff
@@ -4,13 +4,14 @@ let rec detect_list =
...
+    Lwt_unix.file_exists path
+    >>= fun exists -> if exists then return_some (`Ca_file path) else detect_list paths
+;;
 
 let locations =
-  [
-    "/etc/ssl/certs/ca-certificates.crt";
-    "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem";
+  [ "/etc/ssl/certs/ca-certificates.crt"
+  ; "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem"
```
Comment (mikonieminen) [inline lib/ca_certs.ml:13]: I know that this is quite widely used, but what is reasoning behind formatting like this? Personally I find this really weird.

File: lib/ca_certs.mli [hunk 2]
```diff
@@ -1,5 +1,5 @@
 (* TODO delete this once https://github.com/mirage/ca-certs is on opam *)
+
+(** Detect root CAs in the operating system's trust store. Returns [None] if detection did
+    not succeed. The variants correspond to the ones used in [X509_lwt] in [ocaml-tls]. *)
```
Comment (mikonieminen) [inline lib/ca_certs.mli:4]: Does this work correctly with `odoc`? Their documentation advises to add these comments after the function.

**Review by mikonieminen — COMMENTED** [review]
It seems this is build on top of #31 without any reason.

File: lib/ca_certs.ml (hunk 1 above)
Comment (mikonieminen) [reply lib/ca_certs.ml:13]: I mean here that the delimiter is added before next item.

Comment (joseferben) [reply lib/ca_certs.ml:13]: I guess it is more consistent syntactically with ADTs and records.

File: lib/ca_certs.mli (hunk 2 above)
Comment (joseferben) [reply lib/ca_certs.mli:4]: Yes works perfectly with odoc, it's attached to the same AST node.

**Review by mikonieminen — APPROVED** [review]

**Pull request merged**